src/test/resources/alma/(DE-605)HT017411546.json

```diff
@@ -81,6 +81,11 @@
   } ],
   "extent" : "Online-Ressource",
   "note" : [ "Gesehen am 08.10.12", "Beteil. Körp. anfangs: Historische Kommission für Westfalen in Verbindung mit dem Verein für Geschichte und Altertumskunde Westfalens" ],
+  "related" : [ {
+    "label" : "Westfälische Bibliographie zur Geschichte, Landeskunde und Volkskunde",
+    "id" : "http://lobid.org/resources/(DE-605)HT006951721#!",
+    "note" : "Druckausg."
```

Comment:
this id doesn't exist. If you want to reference the Aleph-based lobid API, which is ok, remove the (DE-605) first.

Reply:
you are right, it seems, that I did not properly update the data